From 232cefdc61cf9425f9d30ca71160c9d74587fca7 Mon Sep 17 00:00:00 2001
From: Hillwood Yang <hillwood@opensuse.org>
Date: Mon, 20 Nov 2023 21:43:03 +0800
Subject: [PATCH] fix: Fix unqualified-id error

Fix https://github.com/linuxdeepin/developer-center/issues/6238

Log: It seems that QUICK_ITEM_KEY conflicts with some keywords.
---
 src/dde-dock-plugins/shotstart/shotstartplugin.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/dde-dock-plugins/shotstart/shotstartplugin.cpp b/src/dde-dock-plugins/shotstart/shotstartplugin.cpp
index 699eeb6d..7d5e8815 100755
--- a/src/dde-dock-plugins/shotstart/shotstartplugin.cpp
+++ b/src/dde-dock-plugins/shotstart/shotstartplugin.cpp
@@ -10,7 +10,7 @@
 
 #define ShotShartPlugin "shot-start-plugin"
 #define ShotShartApp "deepin-screen-recorder" // 使用截图录屏的翻译
-const QString QUICK_ITEM_KEY = QStringLiteral("quick_item_key");
+const QString QUICK_ITEM = QStringLiteral("quick_item_key");
 
 ShotStartPlugin::ShotStartPlugin(QObject *parent)
     : QObject(parent), m_iconWidget(nullptr),m_quickPanelWidget(nullptr), m_tipsWidget(nullptr)
@@ -103,7 +103,7 @@ QWidget *ShotStartPlugin::itemWidget(const QString &itemKey)
 //    if (itemKey != ShotShartPlugin) return nullptr;
 
     qInfo() << "Current itemWidget's itemKey: " << itemKey;
-    if(itemKey == QUICK_ITEM_KEY){
+    if(itemKey == QUICK_ITEM){
         qInfo() << "Input Quick Panel Widget!";
         return m_quickPanelWidget.data();
     }else if (itemKey == ShotShartPlugin){
